scripts/py_matter_yamltests/matter_yamltests/hooks.py

```diff
@@ -143,7 +143,7 @@ def step_skipped(self, name: str, expression: str):
         """
         pass
 
-    def step_start(self, name: str):
+    def step_start(self, request):
```

Comment:
please also fix documentation below. Can we also type the request?

Reply:
Done.